packages/network-controller/CHANGELOG.md

```diff
@@ -5,8 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [15.3.0]
 ### Changed
 - Bump dependency `@metamask/eth-query` from ^3.0.1 to ^4.0.0 ([#2028](https://github.com/MetaMask/core/pull/2028))
+- Align `Provider` type to be compatible with `@metamask/eth-query` ([#2028](https://github.com/MetaMask/core/pull/2028))
```

Comment:
This seems more like a fix to me rather than a pure change, and as a result I'm wondering if we want to bump to 15.2.1 rather than 15.3.0 (since we aren't adding anything, which a minor bump would suggest). Thoughts?